pyFV3/stencils/delnflux.py

```diff
@@ -1240,38 +639,40 @@ def __call__(self, q, fx2, fy2, damp_c, d2, mass=None):
         """
 
         if mass is None:
-            self._d2_damp(q, d2, damp_c)
+            self._d2_damp(q=q, d2=d2, damp=damp_c, nord=self._nord)
```

Comment:
Note that some of these damping coefficients are layer-dependent (ie. much stronger and lower-order damping in the sponge layer).